test/sqlsrv/srv_053_mars_disabled_error_checks.phpt

```diff
@@ -0,0 +1,56 @@
+--TEST--
+Error checking for multiple active result sets (MARS) disabled
+--SKIPIF--
+--FILE--
+<?php
+
+require_once("autonomous_setup.php");
+
+// Connect
+$connectionInfo = array("UID"=>$username, "PWD"=>$password, 'MultipleActiveResultSets' => false);
+$conn = sqlsrv_connect($serverName, $connectionInfo) ?: die();
+
+// Query
+$stmt1 = sqlsrv_query( $conn, "SELECT 'ONE'" ) ?: die(print_r( sqlsrv_errors(), true ));
+sqlsrv_fetch( $stmt1 );
+
+// Query. Returns if multiple result sets are disabled
+$stmt2 = sqlsrv_query( $conn, "SELECT 'TWO'" ) ?: die(print_r( sqlsrv_errors(), true ));
+sqlsrv_fetch( $stmt2 );
+
+// Print the data
+$res = [ sqlsrv_get_field($stmt1, 0), sqlsrv_get_field($stmt2, 0) ];
+var_dump($res);
+
+// Free statement and connection resources
+sqlsrv_free_stmt($stmt1);
+sqlsrv_free_stmt($stmt2);
+sqlsrv_close($conn);
+
+print "Done"
+?>
+
+--EXPECT--
+Array
+(
+    [0] => Array
+        (
+            [0] => IMSSP
+            [SQLSTATE] => IMSSP
+            [1] => -44
+            [code] => -44
+            [2] => The connection cannot process this operation because there is a statement with pending results.  To make the connection available for other queries, either fetch all results or cancel or free the statement.  For more information, see the product documentation about the MultipleActiveResultSets connection option.
+            [message] => The connection cannot process this operation because there is a statement with pending results.  To make the connection available for other queries, either fetch all results or cancel or free the statement.  For more information, see the product documentation about the MultipleActiveResultSets connection option.
+        )
+
+    [1] => Array
+        (
+            [0] => HY000
+            [SQLSTATE] => HY000
+            [1] => 0
+            [code] => 0
```

Comment:
Also, I assume the errors come from the second call to sqlsrv_query() not the first sqlsrv_fetch()?